Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -1005,814 +1001,812 @@ function NotNullAssert(v: unknown): asserts v is NotNull {}
 
-    public visible (): boolean {
-        return (this.element && LayoutUtility.IsVisible(this.element)) ? true : false;
-    }
+            this.init();
+            this.element.innerHTML = '';
 
-    public show(message: string): void {
-        NotNullAssert(this.element);
+            const messageDiv = document.createElement("div");
+            messageDiv.innerHTML = message;
 
-        this.init();
-        this.element.innerHTML = '';
+            this.element.appendChild(messageDiv);
+            this.showPopUp();
+        }
+
+        public hide(): void {
+            if (this.repositionTimer) 
+                clearTimeout(this.repositionTimer);
+            this.element && LayoutUtility.Hide(this.element);
+        }
 
-        const messageDiv = document.createElement("div");
-        messageDiv.innerHTML = message;
+        public unbind(): void {
+            this.element && this.element.remove();
+        }
+    };
 
-        this.element.appendChild(messageDiv);
-        this.showPopUp();
-    }
+    export function Selection(field: HTMLInputElement, start: number, end: number, focusInputAfterSelection: boolean): void {
+        if (field.value.length < 2)
+            return;
 
-    public hide(): void {
-        if (this.repositionTimer) 
-            clearTimeout(this.repositionTimer);
-        this.element && LayoutUtility.Hide(this.element);
-    }
+        if (field.setSelectionRange) {
+            field.setSelectionRange(start, end);
+        } else {
+            if (field.selectionStart) {
+                field.selectionStart = start;
+                field.selectionEnd = end;
+            }
+        }
 
-    public unbind(): void {
-        this.element && this.element.remove();
-    }
-};
-Remotion.jQuery.Autocompleter.InformationPopUp = BocAutoCompleteReferenceValue_InformationPopUp;
-
-Remotion.jQuery.Autocompleter.Selection = function(field: HTMLInputElement, start: number, end: number, focusInputAfterSelection: boolean): void {
-    if (field.value.length < 2)
-        return;
-
-    if (field.setSelectionRange) {
-        field.setSelectionRange(start, end);
-    } else {
-        if (field.selectionStart) {
-            field.selectionStart = start;
-            field.selectionEnd = end;
+        if (focusInputAfterSelection) {
+            field.focus();
         }
-    }
+    };
 
-    if (focusInputAfterSelection) {
-        field.focus();
-    }
-};
-
-type BocAutoCompleteReferenceValue_SpaceAround = {
-    top: number;
-    bottom: number;
-    left: number;
-    right: number;
-    spaceVertical: string;
-    spaceHorizontal: string;
-    space: string;
-};
-
-Remotion.jQuery.Autocompleter.calculateSpaceAround = function(element: HTMLElement): BocAutoCompleteReferenceValue_SpaceAround {
-    // re-motion: make sure CSS values are allways numbers, IE can return 'auto'
-    function number(num: number | string) {
-        return parseInt(num as string) || 0;
+    export type SpaceAround = {
```

Comment:
Why do we export those functions? Are they actually used in downstream projects (please check) or is this just an open refactoring to make them private?

Reply:
We export them because we always did, I don't know why we used to do it. There does not seem to be a need for them in our code or downstream, so we could make them private if you want

Reply:
I think we should until there's an actual need. otherwise, we may end up with a dependency we don't want to maintain.